crates/config/src/fuzz.rs

```diff
@@ -127,11 +137,13 @@ mod tests {
         let configs = &[
             "forge-config: default.fuzz.runs = 42424242".to_string(),
             "forge-config: default.fuzz.dictionary-weight = 42".to_string(),
+            "forge-config: default.fuzz.failure-persist-file = fuzz-failure".to_string(),
```

Comment:
Am I correct that this means that by default, each run overwrites the previous? If so I think that's good, to prevent the cache from silently getting too large. We should also make sure an env var args are supported so it's easy to set a one-off name or one-off input file. without modifying the `foundry.toml`

Reply:
Re file persistence:
when Proptest finds a failing test case it writes the case in file and then subsequent runs will first replay those test cases before generating new inputs. So the file is not overwritten and that ensures there's no regression.
Re file size -  I don't think this can grow too much considering that
- once a test fails you have to fix the case in order to get new inputs and possible failures / new lines written in persisted file
- one can use the inline config to redirect failure cases in different files

Re env var: there's `FOUNDRY_FUZZ_FAILURE_PERSIST_FILE` that can be used to specify the input file